### DISCUSSION TITLE
An equivalent for Discord Activities

### DISCUSSION THREAD

**Discussion by VictorVow** [element-hq/element-meta#775] [Ideas]
Discord just announced [Activities](https://discord.com/blog/server-activities-games-voice-watch-together), which allow users in a voice channel to easily watch a YouTube video together, or play a game like Putt Party and Sketch Heads. It would be great for Element to have this feature also.

**Comment by darkralts** [comment]
i agree, that sounds nice and very easy to do.

 for now what i do is use PipeWire with catia which lets you pipe audio applications into your mic, and i screenshare youtube.
